smdebug/tensorflow/utils.py

```diff
@@ -22,7 +22,10 @@ def does_tf_support_mixed_precision_training():
 
 def supported_tf_variables():
     if does_tf_support_mixed_precision_training():
-        from tensorflow.python.keras.mixed_precision.experimental import autocast_variable
+        if is_tf_version_2_4_x():
```

Comment:
You might want to make this if version >= 2.4. 
Once an API moves from experimental to mainstream it stays this way in future versions